gradle.properties

```diff
@@ -3,5 +3,5 @@ fiatVersion=1.18.1
 kapt.use.worker.api=true
 korkVersion=7.36.0
 liquibaseTaskPrefix=liquibase
-org.gradle.parallel=true
+#org.gradle.parallel=true
```

Comment:
Did you mean to change this? I only ask because it wasn't mentioned at all in the PR and seems like the kind of thing you might have done for testing and accidentally committed...

(I only noticed it because it caused a merge conflict for an autobump)

Reply:
We had a test behaving strangely only on CI and this was an attempt to isolate it. Going to revert in another PR today.